src/blocks/card.css

```
@@ -0,0 +1,115 @@
+.card {
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+
+  border-radius: 5px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+
+  color: #060b35;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto 40px;
+  object-fit: contain;
+}
+
+.card__name {
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+
+  color: #060b35;
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+
+  font-family: inherit;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #616070;
+}
+
+.card__review {
+  display: flex;
+  margin-bottom: 24px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__review-value {
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+
+  align-items: flex-end;
+
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 16px;
+
+  line-height: 18px;
+}
+
+.card__price-name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.card__price-value {
+  line-height: 18px;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__link {
+  margin-bottom: 16px;
+  justify-content: center;
+}
+
+.card__item {
+  display: block;
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  color: #fff;
+  background: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  transition: background-color 0.3s linear;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```